src/utils/webrtc/webrtc.js

```diff
@@ -1633,12 +1640,18 @@ export default function initWebRtc(signaling, _callParticipantCollection, _local
 		}
 	})
 
+	// Send the speaking status events via data channel
 	webrtc.on('speaking', function() {
 		sendDataChannelToAll('status', 'speaking')
+		SignalingSpeakingHandler.setSpeaking({
+			speaking: true,
+		})
 	})
-
 	webrtc.on('stoppedSpeaking', function() {
 		sendDataChannelToAll('status', 'stoppedSpeaking')
+		SignalingSpeakingHandler.setSpeaking({
+			speaking: false,
+		})
```

Comment:
[The `LocalMediaModel` provides a `speaking` attribute](https://github.com/nextcloud/spreed/blob/8279dd40388e8c5e93fb62e152a68001fa31e9c3/src/utils/webrtc/models/LocalMediaModel.js#L39) and, therefore, `change:speaking` events; it is the local equivalent of the `speaking` events in `CallParticipantModel`*, so they should be listened instead of explicitly calling the `SignalingSpeakingHandler` from _webrtc.js_ (although in this case the `change:speaking` event from `LocalMediaModel` can be subscribed and unsubscribed when the `SignalingSpeakingHandler` is created and destroyed, as [the `LocalMediaModel` always exists](https://github.com/nextcloud/spreed/blob/8279dd40388e8c5e93fb62e152a68001fa31e9c3/src/utils/webrtc/index.js#L45)).

*Although media and call status for remote participants is unified in `CallParticipantModel` the model for the local participant is split in `LocalMediaModel` and `LocalCallParticipantModel`, because in this case the local media is independent from whether the local participant is in the call or not (for example, for the device preview).